internal/image/resource_image.go

```diff
@@ -728,3 +909,38 @@ func splitImageResourceID(id string) (string, string) {
 	pieces := strings.SplitN(id, ":", 2)
 	return pieces[0], pieces[1]
 }
+
+func exactlyOne(in ...bool) bool {
+	var count int
+	for _, b := range in {
+		if b {
+			count++
+		}
+	}
+	return count == 1
+}
```

Comment:
I see the need for this function also in the implementation for #124. Therefore I wonder, if we should move this function to `internal/utils` package.